### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by pauldelucia**
## Issue being fixed or feature implemented

Update readme for connecting to mainnet, testnet, and local network, and also flesh out the tutorial.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [ ] I have made corresponding changes to the documentation if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-The TUI can connect to any instance of a Dash Platform network, including the testnet, devnets, local networks, and soon, the mainnet. However, for now this readme will only cover connecting to a testnet, and steps to connect to a local network will be added soon. Further steps on running strategies, read [Paul Delucia's guide](https://www.dash.org/blog/strategy-tests-usage-guide/)
+The TUI can connect to any instance of a Dash Platform network, including the mainnet, testnet, devnets, and local networks. The main Install section of this readme covers connecting to mainnet and testnet, with a brief explanation of how to connect to a local network to follow. Check out the TUI documentation [here](https://docs.dash.org/projects/platform/en/1.0.0/docs/tutorials/tui/index.html), and for further steps on running strategy tests, read [Paul DeLucia's guide](https://www.dash.org/blog/strategy-tests-usage-guide/).
```
Comment (thephez) [inline README.md]:
```suggestion
The TUI can connect to any instance of a Dash Platform network, including the mainnet, testnet, devnets, and local networks. The main [Install section](#install) of this readme covers connecting to mainnet and testnet, with a brief explanation of how to connect to a local network to follow. Check out the [TUI documentation](https://docs.dash.org/projects/platform/en/stable/docs/tutorials/tui/index.html), and for further steps on running strategy tests, read [Paul DeLucia's guide](https://www.dash.org/blog/strategy-tests-usage-guide/).
```

File: README.md [hunk 2]
```diff
@@ -32,10 +33,15 @@ Clone the [TUI repo](https://github.com/dashpay/platform-tui):
 git clone https://github.com/dashpay/platform-tui.git
 ```
 
-Open the TUI repo in your terminal and install Rust
+Open the TUI repo in your terminal
 
 ```shell
 cd rs-platform-explorer`
+```
```
Comment (thephez) [inline README.md]:
```suggestion
Open the TUI repo in your terminal:

```shell
cd rs-platform-explorer
```
```

File: README.md [hunk 3]
```diff
@@ -32,10 +33,15 @@ Clone the [TUI repo](https://github.com/dashpay/platform-tui):
...
+
+Install Rust if you don't already have it
```
Comment (thephez) [inline README.md]:
```suggestion
Install Rust if you don't already have it:
```

File: README.md [hunk 4]
```diff
@@ -84,16 +91,24 @@ Do `cargo run` to start the TUI:
...
+* Refresh the wallet balance.
+* If your wallet is not yet funded, you'll need to send it some Dash before you can register an identity. After sending funds, you'll need to refresh the wallet balance again until the funds appear.
+* Now go back to the Main screen and then to the Identities screen.
+* Here, you can either load an existing identity if you have its ID and private keys, or register a new one.
+* If you register a new one, fund it with 0.01 DASH. This should be more than enough to register a (uncontested) DPNS username. The private keys of the newly created identity will be logged to `platform-tui/supporting_files/new_identity_private_keys.log`. Be sure to save a copy of that file somewhere safe.
+* After registering or loading an identity, refresh the identity balance until you see the funds appear.
+* Now, you can register a DPNS username for the identity.
+* After registering the username from the Identities screen, you can navigate back to the Main screen, then the DPNS screen, and from there, query the names for the selected identity, and you should see the name there that you just registered, fetched from the Dash Platform state.
```
Comment (thephez) [inline README.md]: A little tough to digest as a single sentence
```suggestion
* After registering the username from the Identities screen, you can navigate back to the Main screen, then the DPNS screen. From there, query the names for the selected identity, and you should see the name you just registered, which is fetched from the Dash Platform state.
```

**Review by thephez — COMMENTED** [review]
A few suggestions...

**Review by fominok — APPROVED** [review]

**Pull request merged**